build-system/tasks/check-links.js

```diff
@@ -123,6 +134,7 @@ function runLinkChecker(markdownFile) {
   }
   var markdown = fs.readFileSync(markdownFile).toString();
   var filteredMarkdown = filterLocalhostLinks(markdown);
+  filteredMarkdown = filterScriptTagLinks(filteredMarkdown);
```

Comment:
I don't think it would work if we filtered all https:// links that are not on github.com, since there are several genuine links. For example, https://wiki.saucelabs.com/display/DOCS/Setting+Up+Sauce+Connect+Proxy.